lib/block_view_types.go

```diff
@@ -111,19 +112,28 @@ const (
 	EncoderTypeDmThreadEntry                     EncoderType = 37
 	EncoderTypeDeSoNonce                         EncoderType = 38
 	EncoderTypeTransactorNonceEntry              EncoderType = 39
+	EncoderTypeValidatorEntry                    EncoderType = 40
+	EncoderTypeStakeEntry                        EncoderType = 41
+	EncoderTypeLockedStakeEntry                  EncoderType = 42
+	EncoderTypeEpochEntry                        EncoderType = 43
+	EncoderTypeLockedBalanceEntry                EncoderType = 44
+	EncoderTypeLockupYieldCurvePoint             EncoderType = 45
+
 	// EncoderTypeStateChangeEntry represents a state change to a DeSo encoder entry.
-	EncoderTypeStateChangeEntry EncoderType = 40
+	EncoderTypeStateChangeEntry EncoderType = 46
```

Comment:
We need to fix this before we can merge it in. We don't want to push existing encoder types down further. I'll address in a separate PR.